demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -535,14 +535,27 @@ def _upload_item_by_item(
 
         return True
 
-    def _copy_base_pack_docs(self, destination_path: Path):
+    def _copy_base_pack_docs(
+        self, destination_path: Path, marketplace: MarketplaceVersions
+    ):
         documentation_path = CONTENT_PATH / "Documentation"
         documentation_output = destination_path / "Documentation"
         documentation_output.mkdir(exist_ok=True, parents=True)
-        shutil.copy(
-            documentation_path / "doc-howto.json",
-            documentation_output / "doc-howto.json",
-        )
+        if (documentation_path / f"doc-howto-{marketplace.value}.json").exists():
+            shutil.copy(
+                documentation_path / f"doc-howto-{marketplace.value}.json",
+                documentation_output / "doc-howto.json",
+            )
+        elif (documentation_path / "doc-howto-xsoar.json").exists():
+            shutil.copy(
+                documentation_path / "doc-howto-xsoar.json",
+                documentation_output / "doc-howto.json",
```

Comment:
What is doc-howto-xsoar.json? in your content PR you haven't changed the name of the original file

Reply:
This is done so the demisto-sdk PR could be merged independently of the Content PR.